server/data/migrations/20190822201536-update-issue-repositoryId-column.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.sequelize.transaction((/* transaction */) => Promise.all([
+    queryInterface.removeColumn('issues', 'repositoryId'),
+    queryInterface.addColumn('issues', 'repositoryId', {
```

Comment:
Theoretically, you can update the existing field with addConstraint as described here: https://github.com/sequelize/sequelize/pull/7108

Reply:
I tried, but it is not working in my case

Reply:
Understood, it really looks still experimental in some cases